client/src/main/java/com/streamr/client/rest/Stream.java

```diff
@@ -271,4 +272,17 @@ public Stream createStream() {
           lastUpdated);
     }
   }
+
+  public static String createStreamId(String streamIdOrPath, Address owner) {
```

Comment:
@teogeb Is this a good time to extract `StreamId` class?

Reply:
Maybe if we want to add the support for path-only streamIds to all endpoints in the future. Now it is only for `createStream()`.